### PR TITLE
Databricks and Postgres Reverse ETL setup guides

### PR DESCRIPTION
**Pull request by stayseesong**

### CODE REVIEW

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/postgres-setup.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+2. Configure the correction network and security settings for your Postgres database. 
+    * If you're using RDS Postgres, follow [this guide](/docs/connections/storage/catalog/postgres/#network-permissions-for-segment-to-rds). 
+    * Make sure [the following IP addresses](/docs/connections/storage/warehouses/faq/#which-ips-should-i-allowlist) can access the database. 
+3. Run the SQL commands below to create a user named `segment`.
+
+    ```ts
+    -- create a user named "segment" that Segment will use when connecting to your Postgres cluster.
+    CREATE USER segment PASSWORD '<enter password here>';
+
+    -- allows the "segment" user to create new schemas on the specified database. (this is the name you chose when provisioning your cluster) GRANT CREATE ON DATABASE "<enter database name here>" TO "segment";
```
Comment (prayansh) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/postgres-setup.md:22]: @stayseesong can we seperate out the comment and statment here
```suggestion
    -- allows the "segment" user to create new schemas on the specified database. (this is the name you chose when provisioning your cluster) 
    GRANT CREATE ON DATABASE "<enter database name here>" TO "segment";
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/postgres-setup.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    ```
+4. Make sure the user has correct access permissions to the database.
+5. Follow the steps listed in the [Add a source](/docs/connections/reverse-etl/#step-1-add-a-source) section to finish adding Postgres as a source. 
+
+## Extra permissions
+* Give the `segment` user read permissions for any resources (databases, schemas, tables) the query needs to access. 
+
+* Give the `segment` user write permissions for the Segment managed schema (`__segment_reverse_etl`), which keeps track of changes to the query results.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/postgres-setup.md:31]: Does the casing matter for the managed schema? Everywhere else it looks like it's in uppercase - not sure if it should be uppercase here, too!

```suggestion
* Give the `segment` user write permissions for the Segment managed schema (`__segment_reverse_etl`), which keeps track of changes to the query results.  
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* If you want to create the schema yourself instead and then give Segment access to it, run:
+
+    ```ts
+    CREATE SCHEMA IF NOT EXISTS __segment_reverse_etl; 
+    GRANT ALL PRIVILEGES ON SCHEMA __segment_reverse_etl TO `<user or service principal you are using to connect to Segment>`;
+    ```
+
+## Set up guide
+
+To set up Databricks as your Reverse ETL source.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md:32]:
```suggestion
To set up Databricks as your Reverse ETL source:
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+1. Log in to your Databricks account.
+2. Navigate to **Workspaces** and select the workspace you want to use. 
+3. Select **SQL** in the main navigation. 
+4. Select **SQL Warehouses** and select the warehouse you want to use. 
+5. Go to the **Connection details** tab.
+6. In a new tab on your browser, go to the Segment app. 
+7. Navigate to **Connections > Sources > Reverse ETL**.
+8. Click **+ Add Reverse ETL source**. 
+9. Select **Databrticks** and click **Add Source**.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md:42]:
```suggestion
9. Select **Databricks** and click **Add Source**. 
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+3. Select **SQL** in the main navigation. 
+4. Select **SQL Warehouses** and select the warehouse you want to use. 
+5. Go to the **Connection details** tab.
+6. In a new tab on your browser, go to the Segment app. 
+7. Navigate to **Connections > Sources > Reverse ETL**.
+8. Click **+ Add Reverse ETL source**. 
+9. Select **Databrticks** and click **Add Source**. 
+10. Enter the configuration settings for your Databricks source. 
+    * Copy the Hostname, Http Path, and Port from the Databricks console from step 5. 
+    * To generate a **Token** follow the steps listed in the Databricks docs [here](https://docs.databricks.com/dev-tools/auth.html#pat). Segment recomends you to give create token with no expiration date by leaving the lifetime field empty when creating it. If you already have a token with an expiration date, just be sure to keep track of the date and renew it on time.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md:45]:
```suggestion
    * To generate a **Token**, follow the steps listed in the [Databricks docs](https://docs.databricks.com/dev-tools/auth.html#pat). Segment recommends you create a token with no expiration date by leaving the lifetime field empty when creating it. If you already have a token with an expiration date, be sure to keep track of the date and renew it on time.     
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+4. Select **SQL Warehouses** and select the warehouse you want to use. 
+5. Go to the **Connection details** tab.
+6. In a new tab on your browser, go to the Segment app. 
+7. Navigate to **Connections > Sources > Reverse ETL**.
+8. Click **+ Add Reverse ETL source**. 
+9. Select **Databrticks** and click **Add Source**. 
+10. Enter the configuration settings for your Databricks source. 
+    * Copy the Hostname, Http Path, and Port from the Databricks console from step 5. 
+    * To generate a **Token** follow the steps listed in the Databricks docs [here](https://docs.databricks.com/dev-tools/auth.html#pat). Segment recomends you to give create token with no expiration date by leaving the lifetime field empty when creating it. If you already have a token with an expiration date, just be sure to keep track of the date and renew it on time.     
+11. Click **Test Connection** to test to see if the connection works. If the connection fails, make sure you have the right permissions and credentials and try again.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md:46]:
```suggestion
11. Click **Test Connection** to see if the connection works. If the connection fails, make sure you have the right permissions and credentials, then try again.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
+---
+title: Databricks Reverse ETL Setup
+---
+
+Set up Databricks as your Reverse ETL source. 
+
+At a high level, when you set up Databricks for Reverse ETL, the configured user needs read permissions for any resources (databases, schemas, tables) the query needs to access. Segment keeps track of changes to your query results with a managed schema (`__SEGMENT_REVERSE_ETL`), which requires the configured user to allow write permissions for that schema.
+
+## Required permissions
+* To grant access to read data from the tables used in the model query run:
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup.md:10]:
```suggestion
* To grant access to read data from the tables used in the model query, run: 
```

**Review by forstisabella — APPROVED** [review]

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/postgres-setup.md (hunk 2 above)
Comment (prayansh) [reply src/connections/reverse-etl/reverse-etl-source-setup-guides/postgres-setup.md:31]: lets keep it consistent across places so we can uppercase here :)

**Pull request merged**